docs/resources/nginx.md.erb

```diff
@@ -0,0 +1,74 @@
+---
+title: The Nginx Resource
+---
+
+# nginx
+
+Use the `nginx` InSpec audit resource to test the fields and validity of nginx.
+
+Nginx resource extracts and exposes data reported by the command 'nginx -V'
+
+## Syntax
+
+An `nginx` InSpec audit resource block extracts configuration settings that should be tested:
+
+    describe nginx do
+      its('setting_name'){ should eq 'value' }
+    end
+
+where
+
+* `'setting_name'` is a configuration setting defined in the NGINX configuration file
+* `{ should eq 'value' }` is the value that is expected
+
+
+## Supported Properties
+
+* `'prefix'`, `'service'`, `'user'`, `'group'`, `'version'`, `'conf_path'`, `'conf_dir'`,
+* `'sbin_path'`, `'modules_path'`, `'error_log_path'`, `'http_log_path'`,
+* `'pid_path'`, `'lock_path'`, `'http_client_body_temp_path'`,
+* `'http_fastcgi_temp_path'`, `'http_uwsgi_temp_path'`,`'http_proxy_temp_path'`
+* `'http_scgi_temp_path'`
+
+
+## Property Examples and Return Types
+
+### version(String)
+
+`version` returns a formatted string of the version of nginx
+
+    describe nginx do
+      its('version'){ should eq '1.12.0' }
+    end
+
+### openssl_version({String})
+
+`openssl_version ` returns a hash with 'version' and 'date' as keys
+
+    describe nginx.openssl_version['date'] do
+      it { should eq '11 Feb 2013' }
```

Comment:
Hey Adam, thank you for the review. I appreciate your pointers.
 As for the use of 'openssl_version.date' the, 'openssl_version' returns a hash but I am unable to reach the date data without using ' nginx.openssl_version['date']'

I really wanted to use the dot syntax, please let me know if you can show my how to implement that feature.

Thank you